server.port=8002
spring.datasource.url=jdbc:mysql://mysql5045.site4now.net:3306/db_a69d44_lverde?allowPublicKeyRetrieval=true&useSSL=false
spring.datasource.username=a69d44_lverde
spring.datasource.password=Lverde21##@
spring.jpa.generate-ddl=true
spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver

#Jpa/Hibernate :
spring.jpa.show-sql=true
spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
spring.jpa.hibernate.ddl-auto = update

springdoc.api-docs.path=/DetallePuntos/v1/swagger

lima.verde.exception.default.message=Ocurrió un error interno.